packages/components/src/components/contract-card/contract-card-items/accumulator-card-body.tsx

```diff
@@ -92,17 +85,10 @@ const AccumulatorCardBody = ({
                     {take_profit ? <Money amount={take_profit} currency={currency} /> : <strong>-</strong>}
                     {is_valid_to_sell && (
                         <ToggleCardDialog
-                            addToast={addToast}
-                            connectWithContractUpdate={connectWithContractUpdate}
                             contract_id={contract_info.contract_id}
-                            current_focus={current_focus}
-                            error_message_alignment={error_message_alignment}
                             getCardLabels={getCardLabels}
-                            getContractById={getContractById}
                             is_accumulator
-                            onMouseLeave={onMouseLeave}
-                            removeToast={removeToast}
-                            setCurrentFocus={setCurrentFocus}
+                            {...toggle_card_dialog_props}
```

Comment:
Spreading all the props that are only passed through for Accumulators and Multipliers. They were already spread in Turbos.